packages/flutter/lib/src/cupertino/sheet.dart

```diff
@@ -29,6 +29,13 @@ const double _kRoundedDeviceCornersThreshold = 20.0;
 // iOS 18.0.
 const double _kTopGapRatio = 0.08;
 
+// The minimum distance (i.e., maximum upward stretch) from the top of the sheet
+// to the top of the screen, as a ratio of total screen height. This value represents
+// how far the sheet can be temporarily pulled upward before snapping back.
+// Determined through visual tuning to feel natural on <device name, i.e iPhone 16 Pro>
```

Comment:
I meant this suggestion as a placeholder. Which device did you simulate when testing this?

Reply:
Thank you for reviviewing,I tested iPhone16 Pro.